File: UnitySTEPImporter/Assets/UnitySTEPImporter/Runtime/Scripts/ModuleAssembly/Mesh/Color.cs
namespace VENTUS.StepImporter.ModuleAssembly.Mesh
{
    public class Color 
	{
		public Color() { }

		public Color(double r, double g, double b, double a)
		{
			this.red = r;
			this.green = g;
			this.blue = b;
			this.alpha = a;
		}

		private double red;
        private double green;
        private double blue ;
        private double alpha;

        public double Red { get { return red; } }
        public double Green { get { return green; }  }
        public double Blue { get { return blue; }  }		
		public double Alpha { get { return alpha; }  }

	}
}

File: GeomKernel/Module_Assembly/src/Color.cpp
#include "Color.h"

namespace ventus
{
  Color::Color()
  {
    mR = 0.;
    mG = 0.;
    mB = 0.;
    mA = 1.;
  }

  Color::Color( double R, double G, double B, double A)
  {
    mR = R;
    mG = G;
    mB = B;
    mA = A;
  }

  Color::~Color() { }

  void Color::setRGBA(double R, double G, double B, double A)
  {
      mR = R;
      mG = G;
      mB = B;
      mA = A;
  }

} // ventus
